src/Core/TileMesh.js

```diff
@@ -41,6 +42,7 @@ class TileMesh extends THREE.Mesh {
 
         this.layerUpdateState = {};
         this.isTileMesh = true;
+        this.isRefreshing = false;
```

Comment:
I prefer use only layer update state for refreshing statement. You could pass force statement in the command.

Reply:
Done